virttest/test_setup.py

```diff
@@ -1528,6 +1520,7 @@ def sr_iov_setup(self):
             utils_misc.log_line(file_name, dmesg)
             self.setup = None
             return True
+        return True
```

Comment:
No need, should be removed.

Reply:
Actually, it is needed.  It will be executed when re_probe is False.

Reply:
Yes, you are right. I will update.